intl/msg_hash_us.h

```diff
@@ -13668,6 +13668,50 @@ MSG_HASH(
    MSG_CHEEVOS_HARDCORE_MODE_ENABLE,
    "Achievements Hardcore Mode Enabled, save state & rewind were disabled."
    )
+MSG_HASH(
+   MSG_CHEEVOS_HARDCORE_MODE_PAUSED,
+   "Hardcore paused. You cannot earn hardcore achievements for %s using %s"
+   )
+MSG_HASH(
+   MSG_CHEEVOS_LOGGED_IN_AS_USER,
+   "RetroAchievements: Logged in as \"%s\"."
+   )
+MSG_HASH(
+   MSG_CHEEVOS_COULD_NOT_ACTIVATE_ACHIEVEMENT,
+   "Could not activate achievement %u \"%s\": %s"
+   )
+MSG_HASH(
+   MSG_CHEEVOS_COULD_NOT_ACTIVATE_LEADERBOARD,
+   "Could not activate leaderboard %u \"%s\": %s"
+   )
+MSG_HASH(
+   MSG_CHEEVOS_GAME_HAS_NO_ACHIEVEMENTS,
+   "This game has no achievements."
+   )
+MSG_HASH(
+    MSG_CHEEVOS_ALL_ACHIEVEMENTS_ACTIVATED,
+    "All %d achievements activated for this session."
+)
+MSG_HASH(
+    MSG_CHEEVOS_NUMBER_ACHIEVEMENTS_UNLOCKED,
+    "You have %d of %d achievements unlocked."
+)
+MSG_HASH(
+    MSG_CHEEVOS_ALL_ACHIEVEMENTS_ACTIVATED_UNSUPPORTED,
+    "All %d achievements activated for this session (%d unsupported)."
+)
+MSG_HASH(
+    MSG_CHEEVOS_NUMBER_ACHIEVEMENTS_UNLOCKED_UNSUPPORTED,
+    "You have %d of %d achievements unlocked (%d unsupported)."
+)
+MSG_HASH(
+    MSG_CHEEVOS_RICH_PRESENCE_PLAYING,
+    "Playing "
+)
```

Comment:
It's probably better to include the placeholder ("Playing %s") and change the string building to use `snprintf`. String concatenation is generally frowned upon for localization.